### PR TITLE
[ENH] methods for shift, first, last, sd, count

### PR DESCRIPTION
**Pull request by samukweku**
WIP #2946 

- Add methods for shift, first, last, sd, count.

### CODE REVIEW

File: src/core/expr/fexpr.cc [hunk 1]
```diff
@@ -579,6 +579,91 @@ DECLARE_METHOD(&PyFExpr::rowsum)
...
+static const char* doc_shift =
+R"(shift()
+--
+
+Equivalent to :func:`dt.shift(self)`.
+)";
+
+oobj PyFExpr::shift(const XArgs&) {
+  auto shiftFn = oobj::import("datatable", "shift");
+  return shiftFn.call({this});
```
Comment (st-pasha) [inline src/core/expr/fexpr.cc:609]: `dt.shift()` takes optional parameter `n` (the amount of shift), so this method should do the same.

Comment (samukweku) [reply src/core/expr/fexpr.cc:609]: Hi @st-pasha haven't had time to look at this due to other constraints ... I dabbled a bit, but couldnt get to include the extra parameter. Could you kindly show me how to do it. Thanks.

Comment (st-pasha) [reply src/core/expr/fexpr.cc]: done

**Review by st-pasha — APPROVED** [review]

**Comment by samukweku** [comment]
@st-pasha thanks! What does the `?` in `args[0]?` signify?

**Comment by st-pasha** [comment]
It's just C syntax for ternary-if condition: `a? b : c` is the same as python's `b if a else c`.

**Pull request merged**